Library/Homebrew/cmd/formulae.sh

```diff
@@ -0,0 +1,11 @@
+#:  * `formulae`
+#:
+#:  Prints a sorted list of locally available formulas including shortnames.
+#:
+
+# shellcheck disable=SC2155
+homebrew-formulae() {
+  local formulae="$(find "$HOMEBREW_REPOSITORY"/Library/Taps -type d \( -name Casks -o -name cmd -o -name .github \) -prune -false -o -name '*rb' | sed 's/\.rb//g' | sed -E 's .*/Taps/(.*)/homebrew- \1/ g' | sed 's /Formula/ / g')"
```

Comment:
```suggestion
  local formulae="$(
    find "$HOMEBREW_REPOSITORY"/Library/Taps -type d \( -name Casks -o -name cmd -o -name .github \) -prune -false -o -name '*\.rb' 
      | sed -E -e 's/\.rb//g' -e 's .*/Taps/(.*)/homebrew-\1/ g' -e 's /Formula/ / g')"
```

Some questions, though: 
- are you sure the `g` is needed every time on these i.e. there's multiple occurrences on a single line?
- would be nice to split this over multiple lines

Reply:
Also: would be good to handle any naming format for the second directory, ideally and not just `homebrew-`. Alternatively: remove directories with a different format.

Reply:
Ah, good call.  That used to be necessary when the string wasn't split into lines but now I don't think it matters.